molecule/default/latest-converge.yml

```
@@ -15,7 +15,7 @@
     nextcloud_apps:
       calendar:
       contacts:
-        source: "https://github.com/nextcloud-releases/contacts/releases/download/v5.2.0/contacts-v5.2.0.tar.gz"
+        source: "https://github.com/nextcloud-releases/contacts/releases/download/v5.3.1/contacts-v5.3.1.tar.gz"
```

Version 5.3.1 is broken in releases. (it was released with capital V5.3.1)
But it's already fixed in 5.3.2
```suggestion
        source: "https://github.com/nextcloud-releases/contacts/releases/download/v5.3.2/contacts-v5.3.2.tar.gz"
```

---

@staticdev I can see that this change disappeared with last commit/push.

---

I think we discovered that at the same time ;)